Gemfile

```
@@ -10,8 +10,8 @@ def location_for(place)
   end
 end
 
-gem 'vanagon', *location_for(ENV['VANAGON_LOCATION'] || '~> 0.26.2')
-gem 'packaging', *location_for(ENV['PACKAGING_LOCATION'] || '>= 0.106.2')
+gem 'vanagon', *location_for(ENV['VANAGON_LOCATION'] || '~> 0.22.0')
+gem 'packaging', *location_for(ENV['PACKAGING_LOCATION'] || '>= 0.99.76')
```

I think this would still pull in the latest packaging release...I'd try : 
```suggestion
gem 'packaging', *location_for(ENV['PACKAGING_LOCATION'] || '0.106.1')
```